src/TestData/Factories/DonationFormFactory.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace GiveTestData\TestData\Factories;
+
+use GiveTestData\TestData\Framework\Factory;
+
+/**
+ * Class DonationFormFactory
+ * @package GiveTestData\TestData\Factories
+ */
+class DonationFormFactory extends Factory {
+
+	/**
+	 * @var bool
+	 */
+	private $donationGoal;
+
+	/**
+	 * @var bool
+	 */
+	private $termsAndConditions;
+
+	/**
+	 * @var string
+	 */
+	private $template;
+
+	/**
+	 * @var string[]
+	 */
+	private $templates = [ 'sequoia', 'legacy' ];
```

Comment:
This could be extracted to a new provider, which would encapsulate the logic. Then the Donation Form Factory class would not need to change when a new form template is added.

Reply:
I agree.